docs/chipflow-toml-guide.rst

```diff
@@ -0,0 +1,100 @@
+Intro to chipflow.toml
+======================
+
+The ``chipflow.toml`` file provides configuration for your design with the ChipFlow platform.
+
+Let's start with a typical example:
+
+.. literalinclude:: example-chipflow.toml
+   :language: toml
+
+.. testcode::
+   :hide:
+
+   # Assert that example-chipflow.toml matches the current config schema. If 
+   # this test fails, then its likely that the content in this file will need
+   # to be updated.
+   from chipflow_lib.cli import _parse_config_file
+   _parse_config_file("docs/example-chipflow.toml")
+
+project_id
+===========
+
+The ``project_id`` is set to the project ID which you can get from the ChipFlow app project page.
+
+steps
+=====
+
+The ``steps`` define the Python class which will be used as an entry point to these parts of the ChipFlow process.
+You probably won't need to change these if you're starting from an example repository.
+
+silicon
+=======
+
+The ``silicon`` section sets the Foundry ``process`` we are targetting for manufacturing, and the physical ``pad_ring`` we want to place our design inside.
+You'll choose the ``process`` and ``pad_ring`` based in the requirements of your design. 
+
+Available processes
+-------------------
+
++-----------+------------+---------------------------+
+|| Process  || Supported || Notes                    |
+||          || pad rings ||                          |
++===========+============+===========================+
+| sky130    | caravel    | Skywater 130nm            |
++-----------+------------+---------------------------+
+| gf180     | caravel    | GlobalFoundries 130nm     |
```

Comment:
nitpick - 180nm

Reply:
How is that a nitpick? It's flat out incorrect information...